zbus_names/src/bus_name.rs

```diff
@@ -120,6 +120,12 @@ impl Display for BusName<'_> {
     }
 }
 
+impl Debug for BusName<'_> {
```

Comment:
The commit log (`manually implement Debug for *Name`) doesn't mention the rationale. Manual impl is an (no pun intended) implementation detail, not the summary of the change. How about:

```
zn: Improved debug repr for *Name

Improve debug representation of *Name through a manual impl of `Debug`.

Before:

<output>

After:

<output>
```

Reply:
similar for the other Debug impl commits.

Reply:
I think "transparent impl" is a good term to explain this change?

Reply:
Perhaps? or go with my suggestion.

Reply:
Should I repeat it in each commit msg?

Reply:
> Should I repeat it in each commit msg?

repeating is fine.

Reply:
**atomic** commits.